src/dosbox.cpp

```diff
@@ -798,36 +798,38 @@ void DOSBOX_Init() {
 	Pstring = secprop->Add_string("sb_filter", when_idle, "auto");
 	Pstring->Set_help(
 	        "Type of filter to emulate for the Sound Blaster digital sound output:\n"
-	        "  auto:    Use the appropriate filter determined by 'sbtype' (default).\n"
+	        "  auto:  Use the appropriate filter determined by 'sbtype' (default).\n"
 	        "  sb1, sb2, sbpro1, sbpro2, sb16:\n"
 	        "           Use the filter of this Sound Blaster model.\n"
-	        "  none:    Don't filter the output.\n"
+	        "  off:   Don't filter the output.\n"
 	        "A second 'always_on' parameter can be provided to disallow programs from turning the filter off.\n"
 	        "(Example: sbpro1 always_on)");
 
```

Comment:
The conf has a function called `Set_values()` that takes a char * [] of available string options.
Suggest using it for all the filters settings. 
See the `oplmodes` in dosbox.cpp for an example of how its used.

Reply:
The reason why I'm not doing it that way is because at the very end the option to provide completely custom filter settings will be introduced generically for all filters. I'm thinking of the following format:
```
[lp ORDER FREQ] [hp ORDER FREQ]
```

Both are optional, so all the following would be valid:

```
lp 2 4000             # 2nd order LPF @ 4kHz, no HPF
lp 1 8200 hp 4 200    # 1st order LPF @ 8.2kHz, 4th order HPF @ 200Hz
hp 1 500              # 1st order HPF at 500Hz, no LPF
```